meta-sokol-flex-distro/dynamic-layers/virtualization-layer/conf/local.conf.append

```diff
@@ -0,0 +1,3 @@
+# Uncomment the following to enable virtualization support
+#USER_FEATURES += "virtualization"
+#EXTRA_IMAGE_FEATURES += "${@bb.utils.contains('DISTRO_FEATURES', 'virtualization', 'virtualization', '', d)}"
```

Comment:
We should ideally have the second line uncommented by default so the user would only have comment/uncomment the USER_FEATURES just like we do for other features.

Reply:
Ah, good point, thanks!

Reply:
Fixed